docs/development.md

```diff
@@ -40,3 +40,40 @@ See more in the [README.md](www.foia.gov/README.md).
 To build for production, set the `NODE_ENV` and `APP_ENV` environment variables.
 
     $ NODE_ENV=production APP_ENV=production make build
+
+
+## Dev Environment
+
+The new `make dev` command will spin up a dev environment using port `4000` by running Webpack and node in parallel. 
```

Comment:
@ameshkin I think this needs to be updated - maybe "The `make serve` command spins up a..."?